azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java

```diff
@@ -30,12 +30,12 @@
 import org.apache.log4j.Logger;
 
 /**
- * This class is used for creating DB tables by specifying where the create scripts are. The
+ * Creating DB tables by specifying where the create scripts are. The
```

Comment:
change to "Creates"

How about just "Creates DB tables."?

Reply:
Modified as suggested.